### ISSUE TITLE
When I run the code to get traveltime at any grid point, the python kernel always died, why?

### ISSUE THREAD

**Issue by David-zw** [keurfonluu/fteikpy#16]
**Description of the problem**

When I run the example,  the kernel will die at the last line (see the figure below);

**Full code that generated the error**
![image](https://user-images.githubusercontent.com/12843072/153119172-ffb14641-5b5a-4a40-b4fe-5cbc2b825fb4.png)

```python
import numpy
from fteikpy import Eikonal3D

# Velocity model
velocity_model = numpy.ones((8, 8, 8))
dz, dx, dy = 1.0, 1.0, 1.0

# Solve Eikonal at source
eik = Eikonal3D(velocity_model, gridsize=(dz, dx, dy))
tt = eik.solve((0.0, 0.0, 0.0))

# Get traveltime at specific grid point
t1 = tt[0, 1, 2]

# Or get traveltime at any point in the grid
t2 = tt(numpy.random.rand(3) * 7.0)
```

**Full error message**

```
Kernel died, restarting
```

**System information**

* Operating system: Windows 10
* Python installation (Anaconda, system, ETS): Anaconda
* Version of Python: 3.8.8
* Version of this package: latest

**Comment by keurfonluu** [comment]
HI @David-zw ,

Weird. I have then same version of Python through Anaconda (Windows 11, though), and the code runs fine.
What's your version of ``numba``? (I believe that ``numba`` is crashing somehow).

Can you also test pasting that code in a script (e.g., "sample.py"), and then run that script from Windows console? (`python sample.py`). You may have to activate your `conda` environment beforehand (`conda activate`).

Thanks!

**Comment by David-zw** [comment]
Hi @keurfonluu,
The version of numba is the latest. The code sample.py runs fine. When I revise the example to a 2D case(see below), it also runs fine. 
 
![image](https://user-images.githubusercontent.com/12843072/153330623-58fdbb29-fdf4-4dc8-ab0a-639f2656e36d.png)

However, when I run it as a 3D case, the kernel will die. The full error message is as follows:
 
'''
Windows fatal exception: code 0xc000001d


Main thread:
Current thread 0x0000adc0 (most recent call first):
  File "C:\Users\Administrator\AppData\Roaming\Python\Python38\site-packages\fteikpy\_grid.py", line 214 in __call__
  File "C:\Users\Administrator\AppData\Local\Temp\ipykernel_29044\3943618005.py", line 2 in <module>
'''

**Comment by keurfonluu** [comment]
Sorry for the delay.

I've updated my Anaconda distribution to the latest, and I have no issue running this script with numba v0.53.0.
Does the 3D code works fine when you run `python sample.py` (where sample.py is the 3D case)?

I can't debug it as long as I am unable to reproduce the error...

**Comment by David-zw** [comment]
Hi @keurfonluu,

    I've reinstalled Anaconda distribution, and the error would disappear if this package installed in the base environment. But, the error would appear  if this package installed in the new created environment. However, I want to install it in my created environment. Can you also test the package in a  new created environment. Thank you.

**System information**
Operating system: Windows 10
Python installation (Anaconda, system, ETS): Anaconda 2.0.3
Version of Python: 3.8.8
Version of this package: latest

**Comment by keurfonluu** [comment]
Thanks for reporting this issue. I've been able to reproduce it. I have pushed a "fix", so please update to v2.3.2: `pip install fteikpy -U`.
I still don't understand why it worked in Anaconda base environment, though.

**Comment by David-zw** [comment]
Thanks very much for your help.
